envs/dandi/managed-jupyterhub.yaml

```diff
@@ -210,6 +227,16 @@ singleuser:
       profile_options:
         image:
           choices:
+            allensdk:
+              default: true
+              display_name: AllenSDK
+              kubespawner_override:
+                image: ${singleuser_image_repo}:${singleuser_image_tag}-allensdk
+            openscope:
+              default: true
+              display_name: OpenScope
+              kubespawner_override:
+                image: ${singleuser_image_repo}:${singleuser_image_tag}-openscope
```

Comment:
Hm not sure why these changes are showing up since I thought we just merged them in with #238.

Reply:
I must have branched from pre-merge main. Fixed on rebase.